### DISCUSSION TITLE
Best way to make rows per user

### DISCUSSION THREAD

**Discussion by ghostdevv** [Budibase/budibase#1721] [Help] [answered]
I am setting up a todo list and I want each user to have their own todo, what is the best way to go about this?

**Comment by shogunpurple** [comment, marked as answer]
Hey @ghostdevv - this part of the documentation should help you. 

https://docs.budibase.com/data/data-types/relationships/example

The best way would be to create todos and have a 1:N relationship between users -> todos.

You can then list a relationship for the current user, by binding that relationship for your user to a data provider, like this:

![Screenshot 2021-06-16 at 21 50 20](https://user-images.githubusercontent.com/11256663/122291550-da991000-ceec-11eb-8ead-4ed69ffb358e.png)

**Reply by ghostdevv** [reply]
Thank you :D
